### PR TITLE
Build and publish container image to ghcr

### PR DESCRIPTION
**Pull request by mallardduck**
## Description

This change adds a GitHub workflow that will build the master branch's docker image and publish it to GitHubs image repo. The change will make using RMT via docker (or even k8s+rancher) even easier than it is right now. In theory if this is accepted an additional (or modification to an existing) docker-compose file could be added which uses all images from public repos.

## Change Type

*Please select the correct option.*

- [ ] **Bug Fix** (a non-breaking change which fixes an issue)
- [X] **New Feature** (a non-breaking change which adds new functionality)
- [ ] **Documentation Update** (a change which only updates documentation)

## Checklist

*Please check off each item if the requirement is met.*

- [X] I have verified that my code follows RMT's coding standards with `rubocop`.
- [X] I have reviewed my own code and believe that it's ready for an external review.
- [X] I have provided comments for any hard-to-understand code.
- [X] I have documented the `MANUAL.md` file with any changes to the user experience.
- [X] RMT's test coverage remains at 100%.

## Other Notes

Overall I think providing "pre-baked" images of RMT would make it way easier to start hosting RMT from within containers. This change would allow that on either on a Docker or k8s based platform with ease. While the proposed change only pushes the image to GitHub's ghcr, images could also be tagged for and pushed to multiple container hubs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,6 +116,13 @@ All in all, the code you might be working on sits as a volume inside of the
 Docker container. Thus, you will be able to code as usual and the Docker
 container will behave as if you were working entirely locally.
 
+#### Published container images
+Deploying RMT via containers is possible, however should not be used in production
+use-cases. Be aware that container images published on GitHub are master (nightly)
+builds of RMT. These are not official releases and are not ready for production.
```
Comment (digitaltom) [inline README.md:122]: Can you add an example command how to run the container?

File: README.md [hunk 2]
```diff
@@ -116,6 +116,13 @@ All in all, the code you might be working on sits as a volume inside of the
...
+
+Tagged releases of the RMT container image can be found on the [openSUSE Registry](https://registry.opensuse.org/).
```
[the remark this answers is not in the document]
Comment (mallardduck) [reply README.md:124]: For sure, I can make a list of the images you'd use to deploy them here.

**Comment by felixsch** [comment]
@mallardduck @digitaltom It seems like the PR got stale. Do you still want the `RMT` container in the Github Registry?

**Comment by mallardduck** [comment]
@felixsch - Might be best that I close this and folks just use the openSUSE registry instead. There they can find stable tagged containers and a `latest` tag that's not necessarily nightly but should be good enough.

**Pull request closed without merging**